manager/controllers/mockup/datacatalog.go

```diff
@@ -19,6 +19,11 @@ import (
 const (
 	dummyResourceName = "xxx"
 	dummyCredentials  = "dummy"
+	kafkaDeserializer = "io.confluent.kafka.serializers.json.KafkaJsonSchemaDeserializer"
+	s3Literal         = "s3"
+	db2Literal        = "db2"
+	kafkaLiteral      = "kafka"
```

Comment:
S3, DB2, Kafka - although I think we have their definition elsewhere
All the rest should not be constants. Maximum, reuse values using a string variable in the code.